article/_posts/2020-10-05-Diva.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: review
+title: "Diva: Domain invariant variational autoencoders"
+tags: VAE adversarial domain-adaptation
+author: "Antoine Théberge"
+cite:
+    authors: "Maximilian Ilse, Jakub M. Tomczak, Christos Louizos, Max Welling"
+    title:   "Diva: Domain invariant variational autoencoders"
+    venue:   "MIDL 2020"
+pdf: "https://arxiv.org/pdf/1905.10427.pdf"
+---
+
+
+![](/article/images/diva/sc01.jpg)
+
+
+# Highlights
+
+
+
+
+The authors are working on data that has **different domain** and **different classes** and their goal is to disentangle the domain representation from the classes representation.  To do so, they built upon the paper [Learning Latent Subspaces in Variational Autoencoders](https://vitalab.github.io/article/2020/02/13/CSVAE.html).
+
+# Methods
+
+
+To do so, the authors learn 3 latent spaces (as opposed to 1 for a normal VAE) : 
+
+* $$z_d$$ the *domain* latent space;
+* $$z_x$$ the *data* latent space;
+* $$z_y$$ the *class* latent space. 
+
+
+Since the goal is to represent the joint distribution $$p(x,y,z_d,z_x,z_y)$$ and that the random variables are conditionally related, they provide in Figure 1 a graphical representation of the generative and inference models.
```

Comment:
Personally, I'd either not include y in the joint distribution they're trying to learn (and say they're learning $$p(x,z_d,z_x,z_y)$$, and y comes after that from the classifier on the z_y latent subspace) or I'd include d in the joint distribution as well (similar to how they formulate their models in Figure 1).